lib/linguist/languages.yml

```diff
@@ -3154,25 +3154,6 @@ Mask:
   - ".mask"
   tm_scope: source.mask
   language_id: 223
-Mathematica:
-  type: programming
-  extensions:
-  - ".mathematica"
```

Comment:
There are actually [a few users of that extension](https://github.com/search?p=1&q=extension%3Amathematica+NOT+djfhdfbfd&type=Code), so I don't think we can remove it as this point.